security_center/faq.rst

```diff
@@ -0,0 +1,58 @@
+Frequently Asked Questions
+==========================
+
+Security issues
+---------------
```

Comment:
I'll remove.

Reply:
I'm adding a 
.. toctree::
   :maxdepth: 1

to the index.rst file as well for that particular section, otherwise we might end up with a long list of Q&A's on the front page. Which in my opinion will clutter down things.